### ISSUE TITLE
Incorrect intersection Y coordinate

### ISSUE THREAD

**Issue by robhawkes** [tmpvar/2d-polygon-self-intersections#1]
This new library successfully picks up the missed intersections I saw with polygon.js, though the returned intersection coordinates are incorrect.

Given the following square polygon:

``` javascript
var poly = [
  [0, 0], // Point 1
  [10, 0], // 2
  [0, 10], // 3
  [10, 10], // 4
  [0, 0] // Back to point 1
]
```

The intersection should be directly in the middle (so `[5, 5]`), as points 2-3 and 4-1 cross over at a diagonal. However, the returned intersection is actually `[5, 0]`.

Have I done something wrong or is the intersection Y coordinate incorrect?

**Comment by tmpvar** [comment]
sorry about that! stupid fat fingers :)

**Comment by tmpvar** [comment]
oh yeah, latest version published: `2d-polygon-self-intersections@1.1.2`

**Comment by robhawkes** [comment]
Works like a charm now, thanks!
